app/templates/notifications.html

```diff
@@ -0,0 +1,25 @@
+{% extends "base.html" %}
+
+{% block head %}
+<title>{{ profile.username }}'s Notification | HackItForward</title>
+{% endblock %}
+
+{% block body %}
+<div class="content">
+  <div class="row">
+    <div class="col-12">
+      <h2 class="header-margin">Notifications</h2>
+    </div>
+    <div class="card col-12" id="challenge-frame">
+      {% for notif in notifications %}
+        {% if notif != notifications[0] and notif.notification.linked_item.__str__() != notifications[0].notification.linked_item.__str__() %}
```

Comment:
Uhh.... what?

Reply:
this just makes it so that you don't have a line above the first item

Reply:
But ```notif.notification.linked_item.__str__() != notifications[0].notification.linked_item.__str__()```?

Reply:
What you're probably looking for is:
```
{% if notif is not notifications[0] %}
```

Reply:
ah ok, ty